docs/ja/features/subscriptions.md

```
@@ -79,44 +84,44 @@ ws://localhost:3000?tenant={tenantName}
+| `AUTH_ENABLED=true`、有効なトークン、他のテナント | ❌ 接続拒否（1008） |
 | `AUTH_ENABLED=true`、super_admin、任意のテナント | ✅ 接続許可 |
 
 ### 接続フロー
 
-1. クライアントが WebSocket URL に接続 (`tenant` クエリパラメータは必須; 認証が有効な場合はトークンも必須)
-2. サーバーがトークンを検証し、テナントアクセス権限を確認 (認証が有効な場合)
-3. トークンに `cnf.jkt` クレーム (DPoP バインドトークン) が含まれている場合、接続は `pending_dpop` 状態になります — クライアントは 5 秒以内に `dpop_bind` メッセージを送信する必要があります (下記の [DPoP バインディング](#dpop-binding-for-websocket) を参照)
-4. サーバーが DynamoDB に接続を記録 (TTL: 2 時間)
+1. クライアントが WebSocket URL に接続（`tenant` クエリパラメータは必須、認証が有効な場合はトークンも必須）
+2. サーバーがトークンを検証し、テナントアクセス権を確認（認証が有効な場合）
+3. トークンに `cnf.jkt` クレーム（DPoP バインドトークン）が含まれている場合、接続は `pending_dpop` 状態になります — クライアントは 5 秒以内に `dpop_bind` メッセージを送信する必要があります（以下の [DPoP バインディング](#dpop-binding-for-websocket) を参照）
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)